src/custom/components/Header/index.tsx

```diff
@@ -3,11 +3,20 @@ import styled from 'styled-components'
 export { NETWORK_LABELS } from './HeaderMod'
 
 export const Header = styled(HeaderMod)`
-  border-bottom: 1px solid ${({ theme }) => theme.border};
+  border-bottom: 1px solid ${({ theme }) => theme.header?.border};
```

Comment:
what is the fallback border-bottom colour? are we ok with these props being `optional`?

Reply:
The fallback is no border. If no border provided it simply won't output? Which in case that is ok.